checker/http/http.go

```diff
@@ -150,17 +155,42 @@ func WithNoRedirect(noRedirect bool) Option {
 	}
 }
 
+// WithCAFile configures CA file
+func WithCAFile(path string) Option {
+	return func(h *HTTP) {
+		h.caFile = path
+	}
+}
+
+// WithCertFile configures CA file
+func WithCertFile(path string) Option {
+	return func(h *HTTP) {
+		h.certFile = path
+	}
+}
+
+// WithKeyFile configures CA file
+func WithKeyFile(path string) Option {
+	return func(h *HTTP) {
+		h.keyFile = path
+	}
+}
+
```

Comment:
@mortymacs The documentation for all of these functions is the same. Please write specific and relevant documentation for each function.